app/controllers/buyers/applications_controller.rb

```diff
@@ -12,7 +12,8 @@ class Buyers::ApplicationsController < FrontendController
   before_action :find_cinstance, :except => [:index, :create, :new]
   before_action :find_provider,  only: %i[new create update]
 
-  before_action :find_application_plan,          :only => :create
+  before_action :find_application_plan, only: :create
+  before_action :accessible_plans,      only: :index
```

Comment:
You don't need this.

```suggestion
```

Reply:
I added it because `@application_plans` is used in the view. Is there a more preferable way to do it?